### PR TITLE
Add loadfeedback

### PR DESCRIPTION
**Pull request by sebomoto**
Should fix issue #9437 Loading feedback on finishing setup

### CODE REVIEW

File: core/setup/controller.php [hunk 1]
```diff
@@ -147,7 +147,7 @@ public function getSystemInfo() {
 		return array(
 			'hasSQLite' => isset($databases['sqlite']),
 			'hasMySQL' => isset($databases['mysql']),
-			'hasPostgreSQL' => isset($databases['postgre']),
+			'hasPostgreSQL' => isset($databases['pgsql']),
```
Comment (PVince81) [inline core/setup/controller.php]: @sebomoto can you remove this ?

Comment (jancborchardt) [inline core/setup/controller.php]: @sebomoto can you remove that change? It’s unrelated to the rest of this pull request. :)

Comment (sebomoto) [inline core/setup/controller.php]: sorry guys for being unresponsive lately! i want to try to get this done today or tomorrow. to begin with, i don't have a clue why this got included in the pull request. either it happened while i squashed commits or maybe someone else commited to this? will try to figure out what the best way is to remove the change.

Comment (nickvergessen) [inline core/setup/controller.php]: In worst case you can also just make a new PR with the relevant changes

**Comment by sebomoto** [comment]
@jancborchardt @nickvergessen ok - i think i managed to get the mysterious file out of the pull request. seems to have removed part of the conversation here but i hope it doesn't matter too much

**Comment by nickvergessen** [comment]
It just folds the conversion, its okay now

**Comment by jancborchardt** [comment]
Nice, the feedback looks good. Sorry it took so long for the review! :)

@sebomoto you :guitar:!

**Pull request merged**